### DISCUSSION TITLE
Missing shared library libpq.so.5

### DISCUSSION THREAD

**Discussion by dchenk** [diesel-rs/diesel#2808] [Q&A]
When I build my app on the `debian:buster-slim` Docker image and run it I get an `error while loading shared libraries: libpq.so.5: cannot open shared object file: No such file or directory`.

What is my image missing, or is it possible to build the app with this library linked statically?

**Comment by weiznich** [comment]
This error message indicates quite clearly that you miss a package that provides `libpq`. Please search the debian package repository how the package is called there. 
The [Readme of `pq-sys`](https://github.com/sgrif/pq-sys) provides a few more details about different link options.
